src/widgets/list.rs

```diff
@@ -146,6 +149,14 @@ impl<'a> List<'a> {
         self
     }
 
+    /// Set when to show the highlight spacing
+    ///
+    /// See [HighlightSpacing] about which variant affects spacing in which way
```

Comment:
nit:
```suggestion
    /// See [`HighlightSpacing`] about which variant affects spacing in which way
```

Reply:
https://github.com/ratatui-org/ratatui/blob/f63ac72305f80062727d81996f9bdb523e666099/src/widgets/table.rs#L327

Reply:
done for both table and list in 20e988ac1ef462b96e40bfc05995cb251cd34c44